Dockerfile

```diff
@@ -2,14 +2,12 @@ ARG BASE_DOCKER_IMAGE
 
 FROM $BASE_DOCKER_IMAGE
 
-ARG PACKAGES_REPO 
-
 COPY . /src
```

Comment:
Could you also remove that `COPY . /src` command? or is that necessary?

Reply:
Fair, I had not thought of that